automatic/vagrant/tools/chocolateyinstall.ps1

```diff
@@ -1,4 +1,6 @@
-﻿$ErrorActionPreference = 'Stop'
+Import-Module "$PSScriptRoot\..\..\..\extensions\chocolatey-core.extension\extensions\Get-EffectiveProxy.ps1"
```

Comment:
Its not done that way, but in nuspec file as `<dependencies>`. See other packages.

Reply:
Thanks for the hint!